_includes/content/v4notice.html

```diff
@@ -1,4 +1,2 @@
-<div class="v3notice" style="text-align:center;">LoopBack 4 is currently in Developer Preview #3 release.
-    </br/>
-    <a href="http://loopback.io/doc/en/lb3/index.html" class="no_icon">LoopBack version 3</a> is the current release.
+<div class="v3notice" style="text-align:center;">LoopBack 4 is currently in GA(General Availability) release.
```

Comment:
Should we use the same text as in https://github.com/strongloop/loopback-next/pull/1795?

```markdown
LoopBack 4 GA (General Availability) has been released in October 2018, read more in [the announcement post](http://strongloop.com/strongblog/loopback-4-ga)
```

I don't mind either way.

Reply:
Actually, I think it will be best to use the same message in all places for consistency. Let's use the text from strongloop/loopback-next#1795 and strongloop/v4.loopback.io#46 please.

Reply:
> Actually, I think it will be best to use the same message in all places for consistency. 

That sounds good 👍